Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/MainAddActionsNavigationPage.xaml.cs

```diff
@@ -15,14 +24,62 @@ namespace Ginger.BusinessFlowsLibNew.AddActionMenu
     /// </summary>
     public partial class MainAddActionsNavigationPage : Page
     {
-        IWindowExplorer WindowExplorerDriver;
         Context mContext;
+        RecordNavPage mRecordPage = null;
+        SharedRepositoryNavPage mSharedRepositoryNavPage = null;
+        POMNavPage mPOMNavPage = null;
+        ActionsLibraryNavPage mActionsLibraryNavPage = null;
+        LiveSpyNavPage mLiveSpyNavPage = null;
+        WindowsExplorerNavPage mWindowsExplorerNavPage = null;
+
         public MainAddActionsNavigationPage(Context context)
         {
             mContext = context;
             InitializeComponent();
+            context.PropertyChanged += Context_PropertyChanged;
             xNavigationBarPnl.Visibility = Visibility.Collapsed;
             xSelectedItemFrame.ContentRendered += NavPnlActionFrame_ContentRendered;
+
+            ePlatformType ePlatformType = (from x in WorkSpace.Instance.Solution.ApplicationPlatforms
```

Comment:
1- why we have this code which control the buttons avilability twice in constractor and again in Context_PropertyChanged? you should have one private method which will do the buttons control and be called from both places
2-why .IsPlatformSupportPOM() effect on the Record button??? it should effect only the inside Record page on POM relevant functionality. the main "Record" button should be enable/disable based on Driver is IRecord or not